islandora_videojs.module

```diff
@@ -40,7 +58,9 @@ function islandora_videojs_preprocess_islandora_videojs(array &$variables) {
   $videojs_path = libraries_get_path("video-js");
   drupal_add_js($videojs_path . "/video.js");
   drupal_add_css($videojs_path . '/video-js.css');
-
+  if (variable_get('islandora_videojs_hls_library', 'FALSE') === FALSE) {
```

Comment:
Took me an embarrassingly long time to find this; so since this is a checkbox in the admin form, the type of this Drupal variable will be an integer, so it will never be `=== FALSE`. Also this is using a explicit type string 'FALSE' as the default, which is not `=== FALSE`. Finally I think this should be checking for TRUE rather than FALSE when deciding to include the javascript file. So something like this:

```php
if (variable_get('islandora_videojs_hls_library', FALSE)) {
 drupal_add_js($videojs_path . "/videojs-contrib-hls.js");
}
```

Since the checkbox value will be either 1 or 0, the PHP rules for checking truthiness should be fine, as well as with the default `FALSE`.